### PR TITLE
new login/registration pages

### PR DESCRIPTION
**Pull request by travis**
roll up into a single commit because Vercel was choking on the original branch here:

https://github.com/mysilio-co/garden/pull/104

the differences between these branches are now minimal, but this one doesn't
make Vercel sad, so we'll go with this

### CODE REVIEW

File: components/PodLogin.jsx [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+            </button>
+            <div className="text-right text-sm text-my-yellow flex-grow mt-3 md:mt-8">
+              Forgot password?
+            </div>
+          </div>
+          <button
+            type="button"
+            className={`text-white text-sm border-t border-b border-opacity-100 border-white rounded-r-sm bg-opacity-0 items-center mt-5 md:mt-8 p-3 md:p-6`}
+          >
+            Login with certificate (WebId-TLS)
```
Comment (ianconsolata) [inline components/PodLogin.jsx:56]: I didn't see "WebId-TLS" on `/login`, `/register`, or `/`, so where is this component used? I also think we can remove this button, I don't think we really care about WebId TLS based login support.

File: tailwind.config.js [hunk 2]
```diff
@@ -53,89 +17,31 @@ module.exports = {
...
         'my-yellow': '#f2b822',
-        'my-purple': '#944c7d',
-
-
-        'white-a10': 'rgba(255, 255, 255, 0.1)',
-        'white-a20': 'rgba(255, 255, 255, 0.2)'
+        'my-purple': '#944c7d'
+      },
+      backgroundImage: {
+        'forest-landscape': 'url("/img/forest-landscape.png")',
```
Comment (ianconsolata) [inline tailwind.config.js:36]: I don't see this file checked in anywhere, should it be?

File: components/PodLogin.jsx [hunk 3]
```diff
@@ -0,0 +1,77 @@
+import { Logo } from './logo';
+import { Formik } from 'formik'
+import { ArrowRight as ArrowRightIcon } from './icons'
+import { Input } from './inputs';
+import { ArrowCircleRight } from './icons'
+import Image from 'next/image';
+import forestImage from '../public/forest-landscape.png';
```
Comment (ianconsolata) [inline components/PodLogin.jsx:7]: This is a different url than the one in tailwind. Should it be?

File: components/PodLogin.jsx (hunk 1 above)
Comment (travis) [reply components/PodLogin.jsx:56]: oh, it's not - only used in the storybook story. might be worth keeping to help us keep the pod host in sync? ie, when we update styles in garden we'll be able to iterate on the HTML/CSS we need to update the Pod host with

**Review by ianconsolata — CHANGES_REQUESTED** [review]
Register page looks great, but based on chatting with Tani I think we want to tweak the login pages a bit more:

- Can we merge /login and / ? Either redirect to /login instead if you are logged out, or just reuse the same components.
- Can we add a link to /register if you don't already have an account?
- Can we support both password and magic link login, kinda like this:

For this last one, I think we can post directly to the pod host, at `https://mysilio.me/login/password` with a username / password for the top part of the login page, and the have links to register / request a magic link underneath. 

I think folks will still sometimes end up at the pod host login because of the auth redirect, but this should keep most of the login flows entirely on Garden.

**Comment by ianconsolata** [comment]
Oh, right, but all of that stuff is already supported on the POD login page, and we were just gonna skin that. Ugh, I hate login flows. Let's table for now and discuss in Product planning tomorrow with Tani.

File: components/PodLogin.jsx (hunk 3 above)
Comment (travis) [reply components/PodLogin.jsx:7]: whoop, no, didn't look at this, good catch!

**Review by ianconsolata — APPROVED** [review]

**Pull request merged**